node/src/effect.rs

```diff
@@ -1058,7 +1059,7 @@ impl<REv> EffectBuilder<REv> {
     }
 
     /// Puts the given block into the linear block store.
-    pub(crate) async fn put_block_to_storage(self, block: Arc<Block>) -> bool
+    pub(crate) async fn put_block_to_storage(self, block: Arc<BlockV2>) -> bool
```

Comment:
I think we should rename the methods here to reflect the renaming of the block types.

Reply:
Function name, variable name and comment clean-up has been done in [this commit](https://github.com/casper-network/casper-node/pull/4213/commits/d52746adfb6bbf17c908e869d6767d655e3ddbc0) in the [follow-up PR](https://github.com/casper-network/casper-node/pull/4213).